components/Footer/Footer.js

```diff
@@ -1,14 +1,18 @@
 import styled from "styled-components";
 import Link from "next/link";
 import HomePageIcon from "../../public/icons/home_page";
+import UserIcon from "../../public/icons/user";
 
 export default function Footer() {
   return (
     <footer>
       <StyledNav>
-        <StyledLink href="/">
+        <StyledLink href="/" aria-label="to home page">
           <HomePageIcon color={`var(--red-vine-color)`} size="30px" />
```

Comment:
Is this component a styled next Image or img ? Then, as far as I know, it would need an alt attribute with a text explaining what a user could see e.g. "house-icon". Or is the alt text in the component?

Reply:
please be rigid. 😆 It's good for me. lol